### DISCUSSION TITLE
Data not appearing after upgrading Angular from 2 to 7

### DISCUSSION THREAD

**Discussion by eduardoolimpiodev** [primefaces/community#90] [PrimeNG]
Hello everyone, how are you? I need help. I recently updated my project from Angular 2 to Angular 7, and I'm using PrimeNG. Some data is not appearing on the screen, but when I add console.log(), the data appears in the console.

I will post a snippet of the code. Could you tell me what's wrong?

Thank you!

My package.json:
"private": true,
  "dependencies": {
    "@angular/animations": "^7.0.0",
    "@angular/common": "^7.0.0",
    "@angular/compiler": "^7.0.0",
    "@angular/core": "^7.0.0",
    "@angular/forms": "^7.0.0",
    "@angular/http": "^7.0.0",
    "@angular/platform-browser": "^7.0.0",
    "@angular/platform-browser-dynamic": "^7.0.0",
    "@angular/platform-server": "^7.0.0",
    "@angular/router": "^7.0.0",
    "@types/minimatch": "^3.0.3",
    "chart.js": "2.1.3",
    "classlist.js": "^1.1.20150312",
    "core-js": "^2.4.1",
    "fullcalendar": "^3.1.0",
    "jquery": "^3.1.1",
    "jsbarcode": "^3.5.8",
    "moment": "^2.17.1",
    "nanoscroller": "0.8.7",
    "primeicons": "^1.0.0-beta.8",
    "primeng": "6.0.0-alpha.1",
    "quill": "^1.1.8",
    "rxjs": "6.3.3",
    "rxjs-compat": "6.0.0",
    "tslib": "^1.9.0",
    "web-animations-js": "^2.2.2",
    "zone.js": "^0.8.29"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.6.0",
    "@angular/cli": "^7.0.0",
    "@angular/compiler-cli": "^7.0.0",
    "@angular/language-service": "7.0.0",
    "@types/jasmine": "~2.5.53",
    "@types/node": "~6.0.6",
    "angular-ide": "^0.9.77",
    "codelyzer": "~3.2.0",
    "jasmine-core": "~2.6.2",
    "jasmine-spec-reporter": "~4.1.0",
    "karma": "~1.7.0",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.1.2",
    "ts-node": "~3.2.0",
    "tslint": "~5.0.0",
    "typescript": "3.1.1"
  }
}

Code:

<p-dialog header="Relatório de Análise" [(visible)]="displayReport" (onBeforeShow)="onBeforeShowReport()" [contentStyle]="{'height': '580px','width':'1280px'}" modal="modal" showEffect="fade" draggable="false" maximizable="true" responsive="true" height="auto" closeOnEscape="true"> 
              <p-dataTable #tabela [value]="getSampleRegistrationsFilteredReport()" exportFilename="relatorio_analise" csvSeparator=";" [scrollable]="true" scrollHeight="400px">            
                <p-column field="Codigo_Barras" header="Código de Barras" [style]="{'text-align':'center','width':'100px'}"></p-column>
                <p-column field="Num_Sigal" header="Num Sigal" [style]="{'text-align':'center','width':'90px'}"></p-column>
                <p-column field="Extra_Rotina" header="EXR" [style]="{'text-align':'center','width':'70px'}"></p-column> 
                <p-column field="Area" header="Área" [style]="{'width':'100px'}"></p-column>        
                <p-column field="Material" header="Material" [style]="{'width':'150px'}"></p-column>
                <p-column field="Local" header="Local" [style]="{'width':'160px'}"></p-column>
                <p-column field="Ponto_Coleta" header="Ponto Coleta" [style]="{'width':'160px'}"></p-column>
                <p-column field="Usuario" header="Usuário" [style]="{'width':'100px'}"></p-column>
                <p-column field="Data_Cadastro" header="Data Cadastro" [style]="{'width':'150px'}"></p-column>                
                <p-column *ngFor="let col of cols" [field]="col.field" [header]="col.header" [style]="{'width':'150px'}"></p-column>                
              </p-dataTable>         
            <footer>
              <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix">
                <button type="button" pButton icon="fa ui-icon-import-export" iconPos="left" (click)="exportDataTable(tabela)" label="Exportar"></button>
                <button type="button" pButton icon="fa fa-close" iconPos="left" (click)="displayReport=false" label="Fechar"></button>
              </div>

**Comment by cetincakiroglu** [comment]
Hi,

Could you please provide a StackBlitz example so we can identify the issue?

**Reply by eduardoolimpiodev** [reply]
I'm sorry, I can't.
Could you guide me?

**Comment by SoyDiego** [comment]
> I'm sorry, I can't. Could you guide me?

Upload your code in https://stackblitz.com/ and share the link provided an maybe we can help.

Thanks

**Comment by cetincakiroglu** [comment]
Hi,

Until your stackblitz example arrives, please match the Angular version with the PrimeNG version and try again. Major versions of Angular and PrimeNG are compatible with each other. This means, you should use PrimeNG 7 with angular 7.
